usr/share/rear/restore/TSM/default/400_restore_with_tsm.sh

```diff
@@ -88,5 +89,16 @@ for num in $TSM_RESTORE_FILESPACE_NUMS ; do
         LogUserOutput "$log_message"
         echo "$log_message" >>"$backup_restore_log_file"
     fi
+}
+
+for num in $TSM_RESTORE_FILESPACE_NUMS ; do
+    if test "$dsmc_parallel" == "true" ; then
```

Comment:
Usually we do not use things like `"$VAR" == "true"` because
since a longer time we have the is_true and is_false functions
see usr/share/rear/lib/global-functions.sh
https://raw.githubusercontent.com/rear/rear/master/usr/share/rear/lib/global-functions.sh
how to use them.

In general have a look at
https://github.com/rear/rear/wiki/Coding-Style